packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -2845,6 +2845,34 @@ class DiagnosticableTreeNode extends DiagnosticableNode<DiagnosticableTree> {
   }
 }
 
+/// [DiagnosticsNode] for an instance of [FlutterErrorDetails].
+class FlutterErrorDetailsNode extends DiagnosticableNode<FlutterErrorDetails> {
+  /// Creates a [FlutterErrorDetailsNode].
+  FlutterErrorDetailsNode({
+    String name,
+    @required FlutterErrorDetails value,
+    @required DiagnosticsTreeStyle style,
+  }) : super(
+         name: name,
+         value: value,
+         style: style,
+       );
+
+  @override
+  DiagnosticPropertiesBuilder get _builder {
```

Comment:
should `_builder` be public, if we're using it? (Seems weird that you'd only be able to create FlutterErrorDetailsNode in this file, rather than, say, in assertions.dart). cc @jacob314

Reply:
I agreed I will go ahead and make this change, let me know if you have other concern @jacob314

Reply:
I would make the getter public but make FlutterErrorDetailsNode private as I can't think of any reason someone should use it ouside of this library.

Reply:
sounds good